app/models/service_template_ansible_playbook.rb

```diff
@@ -134,26 +144,55 @@ def validate_update_config_info(options)
     self.class.send(:validate_config_info, opts)
   end
 
-  def job_template(action)
+  def job_template!(action)
     resource_actions.find_by!(:action => action.to_s.capitalize).configuration_template
   end
 
-  def update_catalog_item(options, auth_user = nil)
-    config_info = validate_update_config_info(options)
-    name = options[:name]
-    description = options[:description]
+  def job_template(action)
+    resource = resource_actions.find_by(:action => action.to_s.capitalize)
```

Comment:
@bzwei - I tried that last week and still received an AR error - tried a few combinations and AR didn't like looking `:try` off a nil